app/src/components/HomePage/ActionCardSmall.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Card, Box, CardHeader, Text } from "@chakra-ui/react";
+
+interface ActionCardSmallProps {
+  onClick: () => void;
+  icon: React.ReactNode;
+  title: string;
+}
+
+const ActionCardSmall: React.FC<ActionCardSmallProps> = ({
+  onClick,
+  icon,
+  title,
+}) => {
+  return (
+    <Card
+      onClick={onClick}
+      shadow="2dp"
+      backgroundColor="base.light"
+      className="h-[100px] hover:shadow-xl cursor-pointer"
+      py={0}
+      px={6}
+    >
+      <Box className="flex items-center h-full">
+        <Box className="flex items-center justify-center h-[48px] w-[48px] rounded-full bg-[#2351DC]">
```

Comment:
Can we use colors from the Chakra theme instead (by using the Chakra properties that can reference theme keys instead of Tailwind utility classes)?